src/openvic-simulation/pop/Pop.hpp

```diff
@@ -30,7 +30,7 @@ namespace OpenVic {
 	struct Pop {
 		friend struct PopManager;
 
-		using pop_size_t = int64_t;
+		using pop_size_t = int32_t;
```

Comment:
I had to change this as fixed_point_t can't handle int64. Also pop size shouldn't exceed int32.